Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+#!groovy
+
+/*******************************************************************************
+ * Copyright (c) 2015-2019 Konduit, Inc.
```

Comment:
Copyright is wrong :) just say Konduit 2019 please

Reply:
@sshepel just a minor not, rest is fine

Reply:
Done